### PR TITLE
add typeOfProducts field

### PR DESCRIPTION
**Pull request by EngrEric**

### CODE REVIEW

File: src/components/signup/components/SignupForm.js [hunk 1]
```diff
@@ -103,7 +103,7 @@ class SignupModal extends React.Component {
                             label={PRODUCT_TYPE}
                         >
                             {
-                                getFieldDecorator('productType', {
+                                getFieldDecorator('typeOfProducts', {
```
Comment (Way2nnadi) [inline src/components/signup/components/SignupForm.js]: Why did you make this change?

File: src/components/signup/components/Signup.js [hunk 2]
```diff
@@ -39,6 +39,7 @@ class Signup extends React.Component {
                 password: values.password,
                 phoneNumber: values.phoneNumber,
                 state: values.state,
+                typeOfProducts:values.typeOfProducts,
```
Comment (Way2nnadi) [inline src/components/signup/components/Signup.js]: okay i see what you did here

**Comment by Way2nnadi** [comment]
Looks good to me (LGTM)

**Comment by Way2nnadi** [comment]
weqwe

**Comment by joshuaobinna** [comment]
looks good to me

**Comment by Simpcyclassy** [comment]
I think this is good to go

**Pull request merged**